.github/CODEOWNERS

```diff
@@ -256,17 +256,9 @@
 
 # PRLabel: %Mgmt
 /sdk/batch/arm-batch/ @qiaozha @MaryGao
-<<<<<<< HEAD
-/sdk/batch/ci.mgmt.yml @qiaozha @MaryGao
 
 # PRLabel: %Mgmt
 /sdk/baremetalinfrastructure/arm-baremetalinfrastructure/ @qiaozha @MaryGao
-/sdk/baremetalinfrastructure/ci.mgmt.yml @qiaozha @MaryGao
-
-# PRLabel: %Mgmt
-/sdk/batchai/arm-batchai/ @qiaozha @MaryGao
```

Comment:
+1

Reply:
this one is deleted in https://github.com/Azure/azure-sdk-for-js/pull/27405, it's a bad path